view/adminhtml/templates/order/view/synced.phtml

```diff
@@ -15,24 +15,47 @@
  * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  */
 
-// @codingStandardsIgnoreFile
-
-?>
-<?php
+/** @var \Taxjar\SalesTax\Block\Adminhtml\Order\View\Synced $block */
 $order = $block->getOrder();
-$orderSyncDate = '';
-
-if ($block->getSyncedAtDate($order)) {
+$synced = $block->getSyncedAtDate($order);
+if ($synced) {
     $orderSyncDate = $block->formatDate(
-        new \DateTime($block->getSyncedAtDate($order)),
+        new \DateTime($synced),
         \IntlDateFormatter::MEDIUM,
         true
     );
 }
 ?>
-<?php if (!empty($orderSyncDate)): ?>
+<?php if (isset($orderSyncDate)):?>
 <tr>
-    <th><?php /* @escapeNotVerified */ echo __('Synced to TaxJar') ?></th>
-    <td><?php /* @escapeNotVerified */ echo $orderSyncDate ?></td>
+    <th><?= /* @escapeNotVerified */ __('Synced to TaxJar') ?></th>
+    <td><?= /* @escapeNotVerified */ $orderSyncDate ?></td>
 </tr>
+<script>
+    require([
+        'jquery',
+    ], function($) {
+         window.syncTransaction = function(orderId) {
+             var syncButton = $('#taxjar_sync');
+             syncButton.prop('disabled', true);
+             syncButton.find('span').text('Syncing...');
+
+             $.ajax({
+                 url: '<?= $block->escapeUrl($block->getUrl('taxjar/transaction/sync')) ?>',
+                 method: 'POST',
+                 data: {
+                     order_id: orderId
+                 }
+             }).done(function(data) {
+                 if (data.result) {
```

Comment:
data.result is never set so even on a successful sync it show the error alert box. I believe this was intended to be data.success.

Reply:
`data.result` was being set, but only set on "successful" requests.